packages/ui/src/components/ChatList.tsx

```diff
@@ -41,6 +41,7 @@ export function ChatList({
       gap: '$s',
       paddingTop: '$l',
       paddingHorizontal: '$l',
+      paddingBottom: '$6xl',
```

Comment:
Will this value be correct across different sized safe areas? And/or if we tweak navbar size? Would prefer to derive both from a `navBarHeight` constant + `insets.bottom` if possible.

Reply:
Makes sense, will do